src/libs/blueprint/conduit_blueprint_mesh.cpp

```diff
@@ -3145,12 +3279,53 @@ mesh::topology::unstructured::verify(const Node &topo,
             {
                 elems_res &= verify_integer_field(protocol, topo_elems, info_elems, "offsets");
             }
+            if(topo_elems["shape"].dtype().is_string() &&
+              (topo_elems["shape"].as_string() == "polygonal" ||
+               topo_elems["shape"].as_string() == "polyhedral"))
+            {
+                elems_res &= verify_integer_field(protocol, topo_elems, info_elems, "sizes");
+                elems_res &= verify_integer_field(protocol, topo_elems, info_elems, "offsets");   
```

Comment:
I don't think that these checks are necessary as they're covered by the one-to-many relationship schema check (i.e. `blueprint::o2mrelation::verify`). It is true that the schema check doesn't require that "sizes" and "offsets" exist and it's fairly senseless for them to be absent in this context, but our abstract specification requires satisfying only one-to-many relationship adherence and I feel like we should stick to the abstract spec as much as possible.

While that check doesn't explicitly require "sizes" and "offsets" variables, the schema doesn't require these either (polygonal/polyhedral are described as compliant with one-to-many relationships)

Reply:
Ah, that makes sense, thank you for clarifying! I did see the optional checking for "sizes" and "offsets" in `o2mrelation::verify`, but wasn't sure if that extended to polygonal/polyhedral.